### ISSUE TITLE
[A11y Compliance] SfCircleIcon should have field for button description

### ISSUE THREAD

**Issue by mayashavin** [DivanteLtd/storefront-ui#111]
Since it's icon button, the image/icon should have `tabindex = -1` to avoid screen reader to read it, `arial-labelledby` to refer to the main button description/label or at least `arial-label` and `role` to describe it. 
If it is an image button, alt should be enforced (`required` by default)
<img width="1000" alt="Screen Shot 2019-05-24 at 10 06 12" src="https://user-images.githubusercontent.com/6650139/58374027-97ebf000-7f3f-11e9-905f-9854e1ebb3ce.png">

**Comment by peoray** [comment]
Hello, I'd like to work on this :slightly_smiling_face:

**Comment by mayashavin** [comment]
@peoray Sure. 
@filrak can you assign this to @peoray please :)

**Comment by peoray** [comment]
Is there still anything left to do here @filrak @mayashavin ?

**Comment by filrak** [comment]
@mayashavin ?

**Comment by mayashavin** [comment]
@peoray Sorry I didn't see your message. Somehow the screenshot attached was for a different issue. Below is the correct screenshot. We still need to:
1. Add `tabindex = -1` to the icon
2. `arial-labelledby` to refer to the main button description/label or at least arial-label and role to describe it.

![Screen Shot 2019-08-23 at 17 36 25](https://user-images.githubusercontent.com/6650139/63600617-94164b00-c5cc-11e9-90d7-19cc0803586a.png)
Ping me on Discord if you need help.

@filrak Thanks for the reminding :)

**Comment by tohanian** [comment]
Can I be assigned this task?

**Comment by filrak** [comment]
Please accept email invite and you can assign yourself as a collaborator ;)
